### ISSUE TITLE
Bug for dynamic learning rates in fromage optimizer

### ISSUE THREAD

**Issue by ESEberhard** [google-deepmind/optax#1323]
```python
 mult = 1 / jnp.sqrt(1 + learning_rate ** 2)   # error when learning rate is a schedule
 return combine.chain(
      transform.scale_by_trust_ratio(min_norm),
      transform.scale_by_learning_rate(learning_rate * mult),
      transform.add_decayed_weights((mult - 1)),
  )
```

**Comment by rdyro** [comment]
Technically the type annotation indicates that the [fromage optimizer](https://github.com/google-deepmind/optax/blob/29c23a8b57236506f75522525a4d7e6df9438508/optax/_src/alias.py#L1086) does NOT support a learning rate schedule, just a scalar.

Would you like it to? Are you interested in contributing a PR?

**Comment by rdyro** [comment]
Fromage should now accept the learning rate schedule after [https://github.com/google-deepmind/optax/pull/1359](https://github.com/google-deepmind/optax/pull/1359) was merged.
